src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,9 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(
+                ProductService.class
+        );
```

Comment:
Casting the result of `injector.getInstance(ProductService.class)` to `ProductService` can lead to a `ClassCastException` if the `Injector` is not correctly configured to return an instance of `ProductService`. Ensure that the `Injector` is properly set up to handle this class.